proprietary/design-tokens/src/brand/rods/border.tokens.json

```diff
@@ -0,0 +1,10 @@
+{
+  "rods": {
+    "border": {
+      "width": {
+        "zero": { "value": 0 },
+        "m": { "value": "{rods.space.scale.1}" }
```

Comment:
Geen idee of dat juist is, als we in een later stadium echt met de design tokens aan de gang gaan, met Robbert/Bryan en Freek hiernaar kijken